src/pages/Faq/FaqModal.js

```diff
@@ -0,0 +1,336 @@
+import React, { useState } from "react";
+import { RiDeleteBin5Line } from "react-icons/ri";
+import { AiOutlineClose } from "react-icons/ai";
+import styled from "styled-components";
+import theme from "styles/theme";
```

Comment:
```suggestion
import React, { useState } from "react";
import styled from "styled-components";
import { RiDeleteBin5Line } from "react-icons/ri";
import { AiOutlineClose } from "react-icons/ai";
import theme from "styles/theme";
```
import 순서 수정해주세요! 일반적인 convention을 따라 순서만 잘 지켜주셔도 가독성이 좋아집니다. 
아래 순서 참고해주세요.

React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일)